diff --git a/libobs/headers/obs/util/util_uint64.h b/libobs/headers/obs/util/util_uint64.h
index 19a9a07a7..a7cfa075a 100644
--- a/libobs/headers/obs/util/util_uint64.h
+++ b/libobs/headers/obs/util/util_uint64.h
@@ -22,7 +22,7 @@
 
 static inline uint64_t util_mul_div64(uint64_t num, uint64_t mul, uint64_t div)
 {
-#if defined(_MSC_VER) && defined(_M_X64) && (_MSC_VER >= 1920)
+#if defined(_MSC_VER) && defined(_M_X64) && (_MSC_VER >= 1920) && 0
 	unsigned __int64 high;
 	const unsigned __int64 low = _umul128(num, mul, &high);
 	unsigned __int64 rem;
